workflow/labels.md

```diff
@@ -0,0 +1,54 @@
+# Labels descriptions
+
+In this part you will find a description of all the labels we use on issues.
+
+## Labels
+
+| Label       | Description                                                        |
+| -----       | -----------                                                        |
+| Don’t merge | While this label is present this pull-request should not be merged |
+| Duplicate   | This issue already exists                                          |
+| Epic        |                                                                    |
+| Invalid     | The maintainers don’t agree with the issue                         |
+| Won’t fix   |                                                                    |
```

Comment:
Should we have this as well as "Invalid"? I feel like there's maybe so much overlap between the two that they could just be used interchangeably. We should choose the one that sounds less mean.